templates/taxi/manufacturer_list.html

```diff
@@ -1,33 +1,35 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Manufacturer List
-    </h1>
+  <h1>Manufacturer List <a style="margin-left: 20px" href="{% url 'taxi:manufacturer-create' %}">+</a></h1>
 
-    {% if manufacturer_list %}
-      <table class="table">
-        <tr>
-          <th>ID</th>
-          <th>Name</th>
-          <th>Country</th>
-        </tr>
+  {% if manufacturer_list %}
+    <table class="table">
+      <tr>
+        <th>ID</th>
+        <th>Name</th>
+        <th>Country</th>
+        <th>UPDATE</th>
+        <th>DELETE</th>
+      </tr>
 
-        {% for manufacturer in manufacturer_list %}
-            <tr>
-              <td>
-                  {{ manufacturer.id }}
-              </td>
-              <td>
-                  {{ manufacturer.name }}
-              </td>
-              <td>
-                  {{ manufacturer.country }}
-              </td>
-          </tr>
-        {% endfor %}
-      </table>
-
-    {% else %}
-        <p>There are no manufacturers in the service.</p>
-    {% endif %}
+      {% for manufacturer in manufacturer_list %}
+        <tr> <!-- Start of a new row for each manufacturer -->
+          <td>{{ manufacturer.id }}</td>
+          <td>{{ manufacturer.name }}</td>
+          <td>{{ manufacturer.country }}</td>
+          <td>
+            <a class="btn btn-success" href="{% url 'taxi:manufacturer-update' pk=manufacturer.id %}"
+               style="color: white;">UPDATE</a>
+          </td>
+          <td>
+            <a class="btn btn-danger" href="{% url 'taxi:manufacturer-delete' pk=manufacturer.id %}"
```

Comment:
The URL name 'manufacturer-delete' should use a hyphen instead of an underscore to separate words, as per the checklist requirement.